### ISSUE TITLE
python crnn_main.py，cpu运行速率高，跳一堆Unicode字符后出现以下提示，然后就没反应了

### ISSUE THREAD

**Issue by JiangKui007** [chineseocr/chinese-ocr#79]
配置完环境后，运行demo.py正常，尝试自己训练，运行python crnn_main.py --cuda --adadelta 

--random_sample后，在这一步卡住了。请问训练集是如何训练的？放在dataline下的`抱抱、包在我身上超`是啥意思？
`, batchSize=128, beta1=0.5, crnn='../pretrain-models/netCRNN.pth', cuda=True, displayInterval=50, experiment='./save_model', imgH=32, imgW=256, keep_ratio=False, lr=0.005, n_test_disp=1000, ngpu=1, nh=256, niter=1000000, random_sample=True, saveInterval=1000, trainroot='../data/lmdb/train', valInterval=50, valroot='../data/lmdb/val', workers=4)
mkdir: cannot create directory ‘./save_model’: File exists
Random Seed:  5756
loading pretrained model from ../pretrain-models/netCRNN.pth
CRNN(
  (cnn): Sequential(
    (conv0): Conv2d (1, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu0): ReLU(inplace)
    (pooling0): MaxPool2d(kernel_size=(2, 2), stride=(2, 2), dilation=(1, 1))
    (conv1): Conv2d (64, 128, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu1): ReLU(inplace)
    (pooling1): MaxPool2d(kernel_size=(2, 2), stride=(2, 2), dilation=(1, 1))
    (conv2): Conv2d (128, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (batchnorm2): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True)
    (relu2): ReLU(inplace)
    (conv3): Conv2d (256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu3): ReLU(inplace)
    (pooling2): MaxPool2d(kernel_size=(2, 2), stride=(2, 1), dilation=(1, 1))
    (conv4): Conv2d (256, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (batchnorm4): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True)
    (relu4): ReLU(inplace)
    (conv5): Conv2d (512, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu5): ReLU(inplace)
    (pooling3): MaxPool2d(kernel_size=(2, 2), stride=(2, 1), dilation=(1, 1))
    (conv6): Conv2d (512, 512, kernel_size=(2, 2), stride=(1, 1))
    (batchnorm6): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True)
    (relu6): ReLU(inplace)
  )
  (rnn): Sequential(
    (0): BidirectionalLSTM(
      (rnn): LSTM(512, 256, bidirectional=True)
      (embedding): Linear(in_features=512, out_features=256)
    )
    (1): BidirectionalLSTM(
      (rnn): LSTM(256, 256, bidirectional=True)
      (embedding): Linear(in_features=512, out_features=5530)
    )
  )
)

`

**Comment by w7851435** [comment]
自己在目录下创建一下 save_model 文件夹
